.github/actions/abort-release-candidate-v1/action.yml

```diff
@@ -0,0 +1,106 @@
+        # Non-monorepos, or non-npm repos, have a package.json file that contains the updated version number
+        if [ -f "lerna.json" ]; then
+          echo "release_candidate_version=$(cat lerna.json | jq -r '.version')" >> $GITHUB_OUTPUT
+        else
+          echo "release_candidate_version=$(cat package.json | jq -r '.version')" >> $GITHUB_OUTPUT
+        fi
+
+    - name: Get release candidate issue url
+      uses: dequelabs/axe-api-team-public/.github/actions/get-release-issue-v1@main
+      id: get-release-issue
+      with:
+        version: ${{ steps.get-release-candidate-version.outputs.release_candidate_version }}
+      env:
+        GH_TOKEN: ${{ inputs.token }}
+
+    - name: Close release candidate issue as `Not Planned`
+      shell: bash
+      run: |
+        gh issue close ${{ steps.get-release-issue.outputs.issue-url }} -r "not planned"
+      env:
+        GH_TOKEN: ${{ inputs.token }}
+
+    - name: Move release candidate issue to `Not Planned` column (Board 66)
+      uses: dequelabs/axe-api-team-public/.github/actions/add-to-board-v1@main
+      with:
+        issue-urls: ${{ steps.get-release-issue.outputs.issue-url }}
+        column-name: 'Not Doing'
+        project-number: '66'
+      env:
+        GH_TOKEN: ${{ inputs.token }}
+
+    - name: Move release candidate issue to `Release Cancelled` column (Board 103)
+      uses: dequelabs/axe-api-team-public/.github/actions/add-to-board-v1@main
+      with:
+        issue-urls: ${{ steps.get-release-issue.outputs.issue-url }}
+        column-name: 'Release Cancelled'
+        project-number: '103'
+      env:
+        GH_TOKEN: ${{ inputs.token }}
+
+    - name: Get release candidate issue url from docs-repo
+      if: ${{ inputs.docs-repo != 'null' }}
+      uses: dequelabs/axe-api-team-public/.github/actions/get-release-issue-v1@main
+      id: get-release-issue-from-docs-repo
+      with:
+        version: ${{ steps.get-release-candidate-version.outputs.release_candidate_version }}
+        owner-and-repo: 'dequelabs/${{ inputs.docs-repo }}'
+      env:
+        GH_TOKEN: ${{ inputs.token }}
+
+    - name: Close release candidate issue as `Not Planned` in docs-repo
+      if: ${{ inputs.docs-repo != 'null' }}
+      shell: bash
+      run: |
+        gh issue close ${{ steps.get-release-issue-from-docs-repo.outputs.issue-url }} -r "not planned"
+      env:
+        GH_TOKEN: ${{ inputs.token }}
+
+    - name: Move release candidate issue to `Not Planned` column (Board 66) in docs-repo
+      if: ${{ inputs.docs-repo != 'null' }}
+      uses: dequelabs/axe-api-team-public/.github/actions/add-to-board-v1@main
+      with:
+        issue-urls: ${{ steps.get-release-issue-from-docs-repo.outputs.issue-url }}
+        column-name: 'Not Doing'
+        project-number: '66'
+      env:
+        GH_TOKEN: ${{ inputs.token }}
+
+    - name: Reset release branch to ${{ inputs.base }}
+      shell: bash
+      run: |
+        git checkout release 
```

Comment:
Won't need to checkout the release branch if you're already on it in the checkout action.